frontend/login.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+
+ <html lang="en"> 
+    <head>
+        <meta charset="utf-8" />
+        <link rel="apple-touch-icon" sizes="76x76" href="./assets/img/apple-icon.png">
+        <link rel="icon" type="image/png" href="./assets/img/favicon.png">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+        <title>
+          Login Page
+        </title>
+        <meta content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0, shrink-to-fit=no'
+          name='viewport' />
+        <!--     Fonts and icons     -->
+        <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700,200" rel="stylesheet" />
+        <link href="https://use.fontawesome.com/releases/v5.0.6/css/all.css" rel="stylesheet">
+        <!-- CSS Files -->
+      
+        <!-- CSS Just for demo purpose, don't include it in your project -->
+        <link href="src/assets/style.css" rel="stylesheet" />
+      </head>
+    <body>
+        <nav role="navigation">
+            <div id="menuToggle">
+              <input type="checkbox" id="menuToggler" />
+              <!-- hamburger-->
+              <span></span>
+              <span></span>
+              <span></span>
+        
+              <div id="nav-expanded" class="nav-items">
+                <div class="nav-brand">
+        
+                </div>
+                <div>Some Stuff</div>
+                <div>Some Stuff</div>
+                <div>Some Stuff</div>
+              </div>
+            </div>
+          </nav>
+
+        <form action="/login" class="box" method="POST">
+          <h1>Login</h1>
+          <input type="text" name="" placeholder="Username">
+          <input type="password" name="" placeholder="Password">
+          <input type="submit" name="" value="Login"  
```

Comment:
```suggestion
          <input type="text" name="username" placeholder="Username">
          <input type="password" name="password" placeholder="Password">
          <input type="password" name="retypePassword" placeholder="Retype Password">
          <button type="submit">Login</button>  
```
We have a `retypePassword` field in our User POJO, and the `name` attribute is what is sent to the backend as the key of the parameter, so we need that too.

Reply:
^^ agree with this tho.

Reply:
Resolved in 2fcf106